### PR TITLE
add-user-endpoint-blast-radius-dashboard

### PR DESCRIPTION
**Pull request by ericanagle1**

### CODE REVIEW

File: src/boards/user-endpoint-blast-radius/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+Publisher: JupiterOne
+---
+
+# User Endpoint Blast Radius
+
+![sample-screenshot](board.png)
+
+Enter a variable such as an ipAddress to visualize a device owners access, full inventory of digital identities, and resource allocation.
```
Comment (erkangz) [inline src/boards/user-endpoint-blast-radius/readme.md:9]: This board is limited to the macAddress of a device.

File: src/boards/user-endpoint-blast-radius/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# User Endpoint Blast Radius
+
+![sample-screenshot](board.png)
+
+Enter a variable such as an ipAddress to visualize a device owners access, full inventory of digital identities, and resource allocation.  
+
+> Prerequisite: 
+>
+> This works best when integrations to different providers and accounts are
```
Comment (erkangz) [inline src/boards/user-endpoint-blast-radius/readme.md:13]: As many as possible related to user endpoint agents and SaaS applications.

**Review by erkangz — CHANGES_REQUESTED** [review]

File: src/boards/user-endpoint-blast-radius/readme.md (hunk 1 above)
Comment (ericanagle1) [reply src/boards/user-endpoint-blast-radius/readme.md]: adjusted

File: src/boards/user-endpoint-blast-radius/readme.md (hunk 2 above)
Comment (ericanagle1) [reply src/boards/user-endpoint-blast-radius/readme.md]: adjusted

**Review by ericanagle1 — COMMENTED** [review]
Adjustments made

File: src/boards/user-endpoint-blast-radius/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+Publisher: JupiterOne
+---
+
+# User Endpoint Blast Radius
+
+![sample-screenshot](board.png)
+
+Enter a macAddress of a device and visualize the device owners access, full inventory of digital identities, and resource allocation.
```
Comment (erkangz) [inline src/boards/user-endpoint-blast-radius/readme.md:9]:
```suggestion
This is an interactive dashboard that will prompt you to enter a macAddress of a device and visualize the device owner's access, full inventory of digital identities, and the resources they have access to.  
```

**Review by erkangz — APPROVED** [review]

**Review by BrendanQuinnJ1 — APPROVED** [review]
Approved

**Pull request merged**